Packs/BluelivThreatContext/Integrations/BluelivThreatContext/BluelivThreatContext.py

```diff
@@ -564,87 +574,90 @@ def blueliv_indicatorIp(client: Client, nameIP, valueIP):
             'ReadableContentsFormat': formats['markdown'],
             'HumanReadable': tableToMarkdown("Blueliv IP info", human),
             'EntryContext': {
-                'threatContext.hasResults': 'true',
-                'indicator.lastSeen': lastSeen,
-                'indicator.risk': risk,
-                'indicator.latitude': latitude,
-                'indicator.longitude': longitude,
-                'indicator.countryId': countryId,
-                'indicator.campaigns': campaigns,
-                'indicator.campaignIds': campaignIds,
-                'indicator.signatures': signatures,
-                'indicator.signatureIds': signatureIds,
-                'indicator.threatActors': threatActors,
-                'indicator.threatActorIds': threatActorIds,
-                'indicator.tags': tags,
-                'indicator.tagIds': tagIds,
-                'indicator.fqdns': fqdns,
-                'indicator.fqdnIds': fqdnIds,
-                'indicator.sparks': sparks,
-                'indicator.sparkIds': sparkIds,
-                'indicator.bots': bots,
-                'indicator.botIds': botIds
+                'BluelivThreatContext.threatContext.hasResults': 'true',
+                'BluelivThreatContext.indicator.lastSeen': lastSeen,
+                'BluelivThreatContext.indicator.risk': risk,
+                'BluelivThreatContext.indicator.latitude': latitude,
+                'BluelivThreatContext.indicator.longitude': longitude,
+                'BluelivThreatContext.indicator.countryId': countryId,
+                'BluelivThreatContext.indicator.campaigns': campaigns,
+                'BluelivThreatContext.indicator.campaignIds': campaignIds,
+                'BluelivThreatContext.indicator.signatures': signatures,
+                'BluelivThreatContext.indicator.signatureIds': signatureIds,
+                'BluelivThreatContext.indicator.threatActors': threatActors,
+                'BluelivThreatContext.indicator.threatActorIds': threatActorIds,
+                'BluelivThreatContext.indicator.tags': tags,
+                'BluelivThreatContext.indicator.tagIds': tagIds,
+                'BluelivThreatContext.indicator.fqdns': fqdns,
+                'BluelivThreatContext.indicator.fqdnIds': fqdnIds,
+                'BluelivThreatContext.indicator.sparks': sparks,
+                'BluelivThreatContext.indicator.sparkIds': sparkIds,
+                'BluelivThreatContext.indicator.bots': bots,
+                'BluelivThreatContext.indicator.botIds': botIds
             }
         })
     else:
         notFound()
 
 
-def blueliv_indicatorFqdn(client: Client, nameFQDN, valueFQDN):
-    if valueFQDN == '0' and nameFQDN == '0':
+def blueliv_indicatorFqdn(client: Client, args):
+    nameFQDN = args.get('FQDN', False)
+    valueFQDN = args.get('FQDN_id', False)
```

Comment:
better to set the default value with the expected type for the argument (empty string).
this will not affect the conditions in the following lines.
```suggestion
    nameFQDN = args.get('FQDN', '')
    valueFQDN = args.get('FQDN_id', '')
```

relevant for all arguments